packages/hdwallet-keepkey-webusb/src/transport.ts

```diff
@@ -1,13 +1,15 @@
 import * as core from "@shapeshiftoss/hdwallet-core";
 import * as keepkey from "@shapeshiftoss/hdwallet-keepkey";
+import * as eventemitter2 from "eventemitter2";
 
-import { VENDOR_ID, WEBUSB_PRODUCT_ID } from "./utils"
+import { VENDOR_ID, WEBUSB_PRODUCT_ID } from "./utils";
 
-export type Device = USBDevice & {serialNumber: string};
-export class TransportDelegate implements keepkey.TransportDelegate {
+export type Device = USBDevice & { serialNumber: string };
+export class TransportDelegate extends eventemitter2.EventEmitter2 implements keepkey.TransportDelegate {
```

Comment:
Eww. I hate adding yet another `eventemitter2.EventEmitter2` dep -- could we just do 
```
try { 
  await thing.connect()
} catch (e) {
  if (!(e instanceof core.ConflictingApp)) throw e
  // trigger UI things
}
```